src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/project.json

```diff
@@ -1,37 +1,30 @@
-{
-  "version": "1.0.0-*",
+﻿{
+  "version": "1.0.0-23719",
   "description": "",
-  "authors": [
-    ""
-  ],
-  "tags": [
-    ""
-  ],
+  "authors": [ "" ],
+  "tags": [ "" ],
   "projectUrl": "",
   "licenseUrl": "",
+
   "dependencies": {
-    "System.Net.Http": "4.0.1-rc2-23712",
-    "System.Net.Sockets": "4.0.11-rc2-23712",
-    "System.ServiceModel.NetTcp": "4.0.1-rc2-23712",
-    "System.ServiceModel.Http": "4.0.11-rc2-23712",
-    "System.ServiceModel.Primitives": "4.1.0-rc2-23712",
-    "System.ServiceModel.Security": "4.0.1-rc2-23712",
-    "System.Console": "4.0.0-rc2-23712",
-    "System.Collections": "4.0.11-rc2-23712",
-    "System.Linq": "4.0.1-rc2-23712",
-    "System.Threading": "4.0.11-rc2-23712",
-    "System.Diagnostics.Debug": "4.0.11-rc2-23712",
-    "System.Collections.Specialized": "4.0.1-rc2-23712",
-    "System.Private.ServiceModel": "4.0.1-rc2-23712",
-    "System.Diagnostics.Tools": "4.0.1-rc2-23712",
-    "System.ServiceModel.Duplex": "4.0.1-rc2-23712"
```

Comment:
This will cause a build break as build.proj will verify dependency versions for 23712. 

Your two options are: 

1. Modify the `build.proj` file to verify for rc2-23719, and at the same time, touch all `project.json`s 
2. Revert the versions to rc2-23712 in this PR, then touch `build.proj` and all `project.json` files in a subsequent PR. 

I recommend option 2.